source/administration/object-management/object-delete.rst

```diff
@@ -0,0 +1,176 @@
+
+Delete operations on a prior version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To delete prior versions of an object, specify the version's UUID.
+You can retrieve the version UUID with :mc-cmd:`mc ls --versions`. 
+When the ``DELETE`` request specifies a ``version-id`` and the user has the correct permissions to delete the object version`, MinIO permanently removes the specified version from the drive.
+
+.. warning::
+
+   Specifying a ``version-id`` in a DELETE operation is irreversible.
+   MinIO removes the specified version from the drive and **cannot** retrieve it.
+
+Delete all versions
+~~~~~~~~~~~~~~~~~~~
+
+Use :mc-cmd:`mc rm --versions` to delete *all* versions of an object.
+This is irreversible.
+
+Lifecycle Management Expiration
+-------------------------------
+
+You can define one or more :ref:`lifecycle management expiration rule(s) <minio-lifecycle-management-create-expiry-rule>` to expire objects after a certain version number count or a certain period of time.
+When more versions exist than the rule specifies, or when a version is older than specified, MinIO permanently removes the object version from the drive.
+
+These rules rely on the :ref:`scanner <minio-concepts-scanner>` to process the rule on the bucket.
+The scanner operates as a lower priority continuous process where ``READ`` and ``WRITE`` actions are preferred.
+Because of this, object versions that meet the requirements for expiration may not immediately be removed from MinIO.
+
+See the :ref:`scanner <minio-concepts-scanner>` page for more details on how the scanner works and configuration options.
+
+``DeleteMarkers`` are their own objects.
+Lifecycle rules can remove ``DeleteMarkers`` that are the only remaining versions of their objects.
+
+Retained Objects
+----------------
+
+MinIO protects objects subject to a :ref:`locking rule <minio-object-locking>` from being overwritten or deleted.
+These rules require that objects be retained until either the rule expires or is removed.
+
+``DELETE`` operations on locked objects without a specified version result in the creation of a `DeleteMarker` for the object.
+However, the object versions themselves are retained as required by the lock.
+
+``DELETE`` operations that specify an object version are subject to the retention rules.
+MinIO protects object versions subject to a lock from being overwritten or deleted until the lock expires or is removed.
+
+Replicated Objects
+------------------
+
+Replication duplicates objects from one location to another.
+MinIO supports replication at the bucket level or the cluster ("site") level.
+
+Delete operations may or may not replicate, depending on the type of replication and how the replication is configured.
+
+Site Replication
+~~~~~~~~~~~~~~~~
+
+For clusters with :ref:`multi-site replication <minio-site-replication-overview>` set up, MinIO replicates all ``delete`` operations performed on any cluster to each of the other clusters in the peer group.
```

Comment:
"Set up" to me feels a bit more informal than the surrounding text. 
```suggestion
For clusters with :ref:`multi-site replication <minio-site-replication-overview>` enabled, MinIO replicates all ``delete`` operations performed on any cluster to each of the other clusters in the peer group.
```

Reply:
Yes, this is better.